.github/workflows/main_template.yml

```diff
@@ -33,9 +33,8 @@ jobs:
         uses: ./.github/workflows/test_template.yml
         with:
             modulepath: ${{ inputs.modulepath }}
-            tests-timeout: "30m"
+            tests-timeout: "60m"
```

Comment:
I think we discussed why this was needed privately, but it's worth nothing here as well 🙏

Reply:
I will try to make an optimization in this PR to keep gnoland test around 20min like on master
I keep you posted